src/base/ComboBox.js

```diff
@@ -339,10 +339,11 @@ class ComboBox extends Base {
   get [template]() {
     const result = super[template];
 
-    // Put an input element and toggle in the source.
+    // Put a label, input element, and toggle in the source.
     const sourceSlot = result.content.querySelector('slot[name="source"]');
     if (sourceSlot) {
       sourceSlot.replaceWith(fragmentFrom.html`
+        <label id="label" part="label" for="input"></label>
```

Comment:
@JanMiksovsky is this the right approach to add a label? For now, I have aria-label working - but for this to work longterm there needs to be a way to have a label within shadow.